lrc_file/LrcFile.py

```diff
@@ -192,6 +194,34 @@ def cpu_result_data(self) -> dict[str, float]:
         """
         return self._cpu_metrics
 
+    @property
+    def cpu_evaluation_data(self) -> dict[str, float]:
+        """
+            Returns CPU metrics with its values used during evaluation.
+        """
+        cpu_results = [
+            result.data["cpu"]
+            for i, result in enumerate(self._data.results)
+            if _is_cpu_measurement_result(result)
+        ]
+        evaluation_data = {}
+
+        for result in self._data.results:
+            if not isinstance(result, BaselineEvaluationResult):
+                continue
+
+            try:
+                agg_cpu_res = result.data["comparisons"][0]["current_result"].utilization
```

Comment:
i think this would work better as an `isinstance` check on the `current_result` object, this should be `CPUMeasurementResults` type object.

Reply:
thanks